### PR TITLE
Don't create a manufacturer select list in advanced search if store doesn't use them

### PR DESCRIPTION
**Pull request by scottcwilson**
Some stores don't use manufacturers.  Don't create a dropdown list in advanced search for manufacturers if this is true.

### CODE REVIEW

File: includes/modules/pages/advanced_search/header_php.php [hunk 1]
```diff
@@ -24,3 +24,8 @@
   $sData['pfrom'] =    (isset($_GET['pfrom']) ? zen_output_string($_GET['pfrom']) : '');
   $sData['pto'] =      (isset($_GET['pto'])   ? zen_output_string($_GET['pto']) : '');
 
+  // check manufacturers 
+  $mfr_query = $db->Execute("SELECT count(*) AS count FROM " . TABLE_MANUFACTURERS);
```
Comment (drbyte) [inline includes/modules/pages/advanced_search/header_php.php:28]: It's more performant to `SELECT id .... LIMIT 1` and check for `->EOF` than to look through all the thousands of mfgs in the db just to count them.

File: includes/templates/template_default/templates/tpl_advanced_search_default.php [hunk 2]
```diff
@@ -37,11 +37,13 @@
 <br class="clearBoth" />
 </fieldset>
 
+<?php if (!isset($skip_manufacturers) || $skip_manufacturers != true) { ?>
 <fieldset class="floatingBox forward">
     <legend><?php echo ENTRY_MANUFACTURERS; ?></legend>
     <?php echo zen_draw_pull_down_menu('manufacturers_id', zen_get_manufacturers(array(array('id' => '', 'text' => TEXT_ALL_MANUFACTURERS)), PRODUCTS_MANUFACTURERS_STATUS), $sData['manufacturers_id'], 'id="searchMfgId" aria-label="' . PLEASE_SELECT . '"'); ?>
```
Comment (drbyte) [inline includes/templates/template_default/templates/tpl_advanced_search_default.php:43]: If this code-block is skipped, does the absence of the entire `manufacturers_id` field create any issues in the processing of the search form-fields?

File: includes/templates/template_default/templates/tpl_advanced_search_default.php [hunk 3]
```diff
@@ -37,11 +37,13 @@
 <br class="clearBoth" />
 </fieldset>
 
+<?php if (!isset($skip_manufacturers) || $skip_manufacturers != true) { ?>
```
Comment (drbyte) [inline includes/templates/template_default/templates/tpl_advanced_search_default.php:40]: Thoughts about using `!empty()` for this instead?

File: includes/templates/template_default/templates/tpl_advanced_search_default.php (hunk 2 above)
Comment (scottcwilson) [reply includes/templates/template_default/templates/tpl_advanced_search_default.php:43]: Nope no issues.

**Pull request merged**